frontend/public/kubevirt/components/utils/lazyRenderer.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import * as _ from 'lodash-es';
+import { inject } from '../../../components/utils';
+
+const DEFAULT_THROTTLE_DELAY = 250; // in ms
+
+/**
+ * Avoids eager re-rendering for frequent redux store changes.
+ *
+ * Assumption: re-render requests are frequent, so skipping a few of them
+ * has no impact on rendered information from user's perspective.
+ *
+ * If that assumption is not met, the use of this component is not recommended.
+ */
+export class LazyRenderer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.enableRendering = _.throttle(() => {
+      this.forceUpdate();
+    }, DEFAULT_THROTTLE_DELAY);
+  }
+
+  shouldComponentUpdate() {
+    this.enableRendering();
+    return false;
+  }
+
```

Comment:
we should also call the cancel on componentWillUnmount()

Reply:
good catch as the callback uses `this`

Reply:
done